src/app/cli/src/coda.ml

```diff
@@ -237,11 +237,11 @@ let daemon logger =
          ~transport:(Logger.Transport.stdout ()) ;
        (* 512MB logrotate max size = 1GB max filesystem usage *)
        let logrotate_max_size = 1024 * 1024 * 512 in
-       Logger.Consumer_registry.register ~id:"raw_persistent"
+       Logger.Consumer_registry.register ~id:"default"
```

Comment:
would `daemon` be a better name than default, since we're using `prover`, `verifier`, indicating their functionality, for the other processes?

Reply:
I am also using `default` for the prover and verifier. The reason is that if someone just want to create a new logger instead of threading the logger argument all the way down to the call site, the newly created logger would still knows where to store the logs. Because I only use the default case.